### ISSUE TITLE
Failure of generating hosts array for specific conditions.

### ISSUE THREAD

**Issue by fubarhouse** [fubarhouse/ansible-role-tester#78]
Reversing the stdout boolean shows the following error (exit 2, return "", nil).

```
INFO[0000] Checking role hosts...                       
 [WARNING]: Unable to parse /etc/ansible/hosts as an inventory source
 [WARNING]: No inventory was parsed, only implicit localhost is available
 [WARNING]: Could not match supplied host pattern, ignoring: all
 [WARNING]: provided hosts list is empty, only localhost is available
```

**Comment by fubarhouse** [comment]
Defaulting to making `localhost` available only when the value(s) is/are empty.

This is basically to convention, availability, what Ansible prefers/recommends.

It's a very safe bet, and allows users to override the value in their playbooks where necessary.

I'll leave this here for a couple of days before merging. Feedback is desired if it's available/offered.

**Comment by issmirnov** [comment]
Neat! Will this ever hit the statement [here](https://sourcegraph.com/github.com/fubarhouse/ansible-role-tester@14dfb7a/-/blob/cmd/full.go#L119)?

**Comment by fubarhouse** [comment]
Yes it will, in fact it probably will get hit more than it previously did.

So running a playbook targeting localhost with the remote flag basically runs it against the local file system. This basically voids it as a test and runs the playbook against the file system you're using.

#66 Ensured that this simply failed with the given error message to prevent unwanted behavior and describing the reasoning. I am however interested in changing the message to be a bit clearer.
